### PR TITLE
erik-pardo/pivotal-projects

### PR DESCRIPTION
**Pull request by epardo-23**
Added features files using the step definitions

### CODE REVIEW

**Comment by epardo23** [comment]
image
Allure Report of Pivotal

**Comment by epardo23** [comment]
As well I added a new commit with my last changes where I fixed some problems with wrong code implemented before.
Please check it and thanks!

File: gradle.properties [hunk 1]
```diff
@@ -1 +1 @@
-cucumberOptions=
+cucumberOptions=--tags @allPivotalProject
```
Comment (marcos-x86) [inline gradle.properties]: Please consider leaving the content of cucumberOptions in blank.

File: config.json.dist [hunk 2]
```diff
@@ -1,27 +0,0 @@
-{
```
Comment (marcos-x86) [inline config.json.dist]: Please consider not deleting this template file.

**Review by marcos-x86 — CHANGES_REQUESTED** [review]

**Comment by marcos-x86** [comment]
Please update your branch and consider not deleting config.json.dist template file.

**Comment by epardo-23** [comment]
I completed trello automated test cases and I sent a new commit with all changes.
Please check it. Thanks
image

**Review by marcos-x86 — APPROVED** [review]

**Pull request merged**